distributed/dashboard/components/shared.py

```diff
@@ -454,12 +456,23 @@ def trigger_update(self):
 
 
 class SystemMonitor(DashboardComponent):
-    def __init__(self, worker, height=150, **kwargs):
+    def __init__(self, worker, height=150, last_count=None, **kwargs):
         self.worker = worker
 
         names = worker.monitor.quantities
-        self.last = 0
+        self.last_count = 0
+        if last_count is not None:
+            names = worker.monitor.range_query(start=last_count)
+            self.last_count = last_count
         self.source = ColumnDataSource({name: [] for name in names})
+        self.label_source = ColumnDataSource(
+            {
+                "x": [5] * 3,
+                "y": [70, 55, 40],
```

Comment:
These coordinates place the labels roughly in the middle of the left side of the plots - is there a better place they could go? I was thinking the top/bottom right corners, but that seemed to block the plot outputs more often.

Reply:
Hmm, that's a good question.  What do you think of adding a table below the plots ?  Does that result in occupying too much space on the page?  I like the idea of inlining the info but perhaps that clutter will cause confusion and/or difficulty reading.  If you don't think so that's ok

Reply:
I personally feel like a table below the plots would be equivalent to having a table in a separate tab, since in either case you'd only be able to look at one or the other (unless you're thinking the tables be directly under the plot they correspond to).

Maybe this info would be suited for a `HoverTool` so that it only pops up when you mouse over the plot?